optee: move optee recipes to common storage

oleksiimoisieiev opened the pull request:
Signed-off-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>

lorc commented:
We have very similar changes in `domd-image-weston`. I would prefer to not have almost similar changes in different images.

It is much better to move the common fixes into some common place and then only make image-specific changes in image recipes

oleksiimoisieiev committed:
optee: Move optee fixes to common storage

Moving optee fixes from domd-image-weston to common layer because
domd-image-minimal needs those fixes as well.

Signed-off-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>
Reviewed-by: Volodymyr Babchuk <volodymyr.babchuk@epam.com>

oleksiimoisieiev committed:
python3: Move pyelftools recipe to common storage

Moving pyelftools recipe, needed by newer version of OP-TEE, to common
layer

Signed-off-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>
Reviewed-by: Volodymyr Babchuk <volodymyr.babchuk@epam.com>

oleksiimoisieiev committed:
optee: remove optee-os_git.bbappend and use from common storage

Removing duplicating optee-os_git.bbappend from domd-image-minimal and
use optee from common storage.

Signed-off-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>
Reviewed-by: Volodymyr Babchuk <volodymyr.babchuk@epam.com>

oleksiimoisieiev commented:
> We have very similar changes in `domd-image-weston`. I would prefer to not have almost similar changes in different images.
> 
> It is much better to move the common fixes into some common place and then only make image-specific changes in image recipes

@lorc Moved optee to the common storage. Please review

lorc reviewed (approved):
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`
